fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -446,6 +446,8 @@
 
     private long feStartTime;
 
+    private WorkGroupMgr workGroupMgr;
```

Comment:
you need to support `loadImage` and `saveImage` methods. otherwise, your metadata will lost.
```
    public void loadImage(String imageDir) throws IOException, DdlException {

    // Only called by checkpoint thread
    public void saveImage() throws IOException {
```

Reply:
ok